src/librustc_codegen_llvm/builder.rs

```diff
@@ -652,6 +652,56 @@ impl BuilderMethods<'a, 'tcx> for Builder<'a, 'll, 'tcx> {
         unsafe { llvm::LLVMBuildSExt(self.llbuilder, val, dest_ty, UNNAMED) }
     }
 
+    fn fptoui_sat(&mut self, val: &'ll Value, dest_ty: &'ll Type) -> Option<&'ll Value> {
+        if self.sess().target.target.arch == "wasm32"
+            && self
+                .sess()
+                .target_features
+                .contains(&rustc_span::symbol::Symbol::intern("nontrapping-fptoint"))
```

Comment:
I think instead of calling `intern` here you'll want to [add to this list](https://github.com/rust-lang/rust/blob/229e5b2640fc5715e77607a989748be588d983f2/src/librustc_span/symbol.rs#L289) and then use `sym::nontrapping_fptoint`

Reply:
Ah, I actually had it in this list, but I didn't see that you could use custom string literals. Yeah I will do that.